### DISCUSSION TITLE
Icy metadata on local icecast missing?

### DISCUSSION THREAD

**Discussion by RM-FM** [AzuraCast/AzuraCast#6714] [General]
Hi,

As I did not found any issue reported on this, I can hardly imagine that is a general issue/bug. That is why I did not report it as an issue for now.

The local radio.mp3 Icecast endpoint does not show any icy metadata (e.g. in VLC player). It works perfectly fine if the same station connects to an external Icecast server.

I spotted this already on earlier Azuacast versions but it is also happening with the latest rolling release docker version (yesterday).

Any idea what could cause that? Is it a bug or did I miss something?

**Comment by gAlleb** [comment]
Hi @RM-FM . Well it is not an azuracast issue (a VLC one perhaps). It is probably due to HTTPS protocol. 
Try and open a stream with HTTP in url instead of HTTPS and you will see metadata.

**Comment by RM-FM** [comment]
@gAlleb Bingo! I knew this can‘t be an actual issue ;-) Thanks for the hint.

**Comment by Moonbase59** [comment]
Just cross-checked, using Rolling Release #[155f66e](https://github.com/AzuraCast/AzuraCast/commit/155f66e0a31d045dff5cfd087bc952e058067af0) (2023-11-05 17:46), and it definitely works:

![Bildschirmfoto vom 2023-11-07 11-51-24](https://github.com/AzuraCast/AzuraCast/assets/3706922/bce74f71-1d89-46dd-87de-adc16e4a7c2d)

The song display comes from my [StudioDisplay](https://github.com/Moonbase59/studiodisplay) telemetry (MQTT) and is directly fed from the in-stream metadata. It was done like this for verification: If the song display goes blank on the studio monitor, something is wrong with the in-stream metadata. :-)

Needs a listener slot, of course:

![Auswahl_298](https://github.com/AzuraCast/AzuraCast/assets/3706922/13ed61b2-db4d-4858-819f-434b5c31c64d)
